Dockerfile.okd

```diff
@@ -0,0 +1,16 @@
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-base-openshift AS builder
+
+COPY ./get-resource.sh /usr/local/bin/get-resource.sh
+
+RUN /usr/local/bin/get-resource.sh
```

Comment:
This command requires an argument - the image URL, which you don't have. You need to look at the upstream version and replicate it here probably: https://github.com/metal3-io/ironic-ipa-downloader/blob/master/get-resource.sh

Reply:
Ah, nevermind, missed the change in resource.sh at first. The previous comment applies though